examples/python/gymnasium_vect_bench.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+
+#####################################################################
+# Test and benchmark the vectorization of VizDoom in gymnasium
+#####################################################################
+
+import time
+import argparse
+import warnings
+
+from gymnasium.vector import AsyncVectorEnv
+import gymnasium
+from vizdoom import gymnasium_wrapper
+
+warnings.filterwarnings("ignore")
+parser = argparse.ArgumentParser()
+parser.add_argument("--n_envs", type =int, default = 1, help ="Number of envs")
+args =parser.parse_args()
+seed = 42
+n_episodes = 1000
+
+# Pick an environment VizdoomCorridor-v0
+envs = gymnasium.make_vec("VizdoomCorridor-v0", num_envs =args.n_envs, vectorization_mode="async")
+
+# Time it
+start = time.time()
+
+observation, info = envs.reset()
+for _ in range(n_episodes):
```

Comment:
These are not really full episodes of the environment, right? I think steps or iterations would be a better name.